services/static-webserver/client/source/class/osparc/cluster/ClustersDetails.js

```diff
@@ -50,8 +50,11 @@ qx.Class.define("osparc.cluster.ClustersDetails", {
       });
       osparc.cluster.Utils.populateClustersSelectBox(selectBox);
       selectBox.addListener("changeSelection", e => {
-        const clusterId = e.getData()[0].id;
-        this.__selectedClusterChanged(clusterId);
+        const selection = e.getData();
+        if (selection.length) {
+          const clusterId = [0].id;
```

Comment:
Is this correct `[0]` ? should it be something like `selection[0].id`?

Reply:
Correct, thanks!